src/utils/apis/tasksAPI.js

```diff
@@ -1,12 +1,13 @@
 import {recruitmentServiceBasicAPILink} from "./APILinks";
 import Swal from 'sweetalert2'
 import {useDevPassword} from "../hooks/useDevPassword";
+import {authFetch} from "../authFetch";
 
 export const tasksApi = {
 
     getTask: (processId) => {
         if(processId){
-            return fetch(`${recruitmentServiceBasicAPILink}/api/tasks?process=${processId}`)
+            return authFetch(`${recruitmentServiceBasicAPILink}/api/tasks?process=${processId}`)
```

Comment:
Jesteś pewny, że tu powinien być authFetch? Nie próbowałbym go zmieniać tam, gdzie jest on nie potrzebny. Nie pamiętam, jak to dokładnie jest, ale jeśli w authFetch JWT jest nienullowalny. to taką zmianą zrobimy sobie kuku i to dość mocne, bo niezalogowany użytkownik nie będzie mógł nic zrobić.